app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,75 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: [:show, :edit, :update, :destroy]
+
+  # GET /posts
+  # GET /posts.json
+  def index
+    @posts = Post.all
+    render template: "posts/index.html.erb", layout: "application"
```

Comment:
I believe this line isn't needed as well, if you have the index file inside the posts folder, Ruby will automatically detect that correlation when you set up the "post resources" inside the Routes file.